src/site/content/en/learn/learn.11tydata.js

```diff
@@ -43,6 +45,7 @@ module.exports = function () {
 
   return {
     learn: {
+      web_dev_basics,
```

Comment:
To avoid keeping this PR too long and geting it to bekome stale, let's comment out this line (to checkin the collection but keep it disabled for now) and merge. wdyt?

Reply:
Just updated some links, but yea this sounds good.